packages/react-dom/src/client/ReactDOMEventHandle.js

```diff
@@ -234,15 +209,7 @@ export function createEventHandle(
       );
       if (!doesTargetHaveEventHandle(target, eventHandle)) {
         addEventHandleToTarget(target, eventHandle);
-        registerReactDOMEvent(
-          target,
-          domEventName,
-          isPassiveListener,
-          isCapturePhaseListener,
-          listenerPriority,
-        );
-        // Add the event to our known event types list.
-        addEventTypeToDispatchConfig(domEventName);
```

Comment:
No need to do this now since we've removed support for arbitrary events for now.